DSharpPlus.Interactivity/InteractivityExtension.cs

```diff
@@ -245,7 +350,7 @@ public async Task<ReadOnlyCollection<Reaction>> CollectReactionsAsync(DiscordMes
 
             var timeout = timeoutoverride ?? this.Config.Timeout;
             var collection = await this.ReactionCollector.CollectAsync(new ReactionCollectRequest(m, timeout)).ConfigureAwait(false);
-            return new ReadOnlyCollection<Reaction>(collection.ToList());
+            return collection.ToList().AsReadOnly();
```

Comment:
`ToList().AsReadOnly()` is completely unnecessary here, it's already the right type.